Tests/Foundation/Tests/TestDecimal.swift

```diff
@@ -7,8 +7,13 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
-import Foundation
-import XCTest
+#if NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT
```

Comment:
This was necessary for TestDecimal to build on macOS against the correct Foundation library.  See also https://github.com/apple/swift-corelibs-foundation/pull/4643/commits/e9e183cf9041841fea1700f5beeebffd0bedd09b#r1011034654 .  I was mistaken in thinking this change was unnecessary.